e2e/node/manager_test.go

```diff
@@ -197,79 +187,60 @@ func (s *ManagerTestSuite) TestNodeStartStop() {
 	s.NoError(err)
 
 	// try stopping non-started node
-	s.False(s.NodeManager.IsNodeRunning())
-	_, err = s.NodeManager.StopNode()
+	err = s.NodeManager.StopNode()
 	s.Equal(err, node.ErrNoRunningNode)
 
 	// start node
-	s.False(s.NodeManager.IsNodeRunning())
-	nodeStarted, err := s.NodeManager.StartNode(nodeConfig)
+	err = s.NodeManager.StartNode(nodeConfig)
 	s.NoError(err)
-	// wait till node is started
-	<-nodeStarted
-	s.True(s.NodeManager.IsNodeRunning())
 
 	// try starting another node (w/o stopping the previously started node)
-	_, err = s.NodeManager.StartNode(nodeConfig)
+	err = s.NodeManager.StartNode(nodeConfig)
 	s.Equal(err, node.ErrNodeExists)
 
 	// now stop node
-	nodeStopped, err := s.NodeManager.StopNode()
+	err = s.NodeManager.StopNode()
 	s.NoError(err)
-	<-nodeStopped
-	s.False(s.NodeManager.IsNodeRunning())
 
 	// start new node with exactly the same config
-	nodeStarted, err = s.NodeManager.StartNode(nodeConfig)
+	err = s.NodeManager.StartNode(nodeConfig)
 	s.NoError(err)
-	// wait till node is started
-	<-nodeStarted
```

Comment:
Why do we not validate that the node has fully started using the `nodeStarted` channel in the new changes?